### PR TITLE
add git-clonerepos.yml to test_ansible.bats script

### PR DESCRIPTION
**Pull request by sometheycallme**

### CODE REVIEW

**Comment by kropped** [comment]
:bell: Staged `cleanerbot_test_gitrepo` at revision 1d7906432a54e32c04a8ce309616a1b7daf4e3e1 on 1b9907cdd369

**Comment by kropped** [comment]
:bell: Staged `cleanerbot_test_gitrepo` at revision 31e6edfec42933d21b9af20546de60adcb91b752 on 1b9907cdd369

File: webserver/server.py [hunk 1]
```diff
@@ -61,9 +61,15 @@ def do_POST(self):
 		    return
 	    directory = post_data['git_handle'] + '/' + post_data['branch_name']
 	    safe_dir = self.alphafy(directory)
+	    path = "{0}/{1}".format(base_dir, safe_dir)
```
Comment (jumanjiman) [inline webserver/server.py]: it's more portable to do:

```
path = os.path.join(base_dir, safe_dir)
```

Comment (sometheycallme) [inline webserver/server.py]: made the modification and tested.  works!

**Pull request merged**